### DISCUSSION TITLE
Versioning

### DISCUSSION THREAD

**Discussion by sdebruyn** [OHDSI/Achilles#561] [General]
Please setup a versioning scheme, both for this package as well as for its dependencies.

**Comment by fdefalco** [comment]
The current release of Achilles is 1.6.3.  There is a draft release for 1.7 that would add CDM 6.0 support.  That being said, CDM 6 is likely going to be revised significantly before release and we will be waiting for that release before adding support.  

The master branch currently has a large number of fixes and will be pushed into a release once testing has been completed.  Pulling from master currently reflects version 1.6.7.

As an R package you can find additional dependency information in the DESCRIPTION file.
